components/User/UserProfile/UserProfile.js

```diff
@@ -143,8 +153,14 @@ class UserProfile extends React.Component {
         return (<PrivatePublicUserProfile user={this.props.UserProfileStore.user} decks={this.props.UserProfileStore.userDecks}/>);
     }
 
+    displayIntegrations() {
```

Comment:
If it's that easy, you might be able to omit the function at all by moving `<Integrations />` upwards.